Realm/Realm/DatabaseTypes/RealmObjectBase.cs

```diff
@@ -453,4 +474,4 @@ public Metadata(TableKey tableKey, IRealmObjectHelper helper, IDictionary<string
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
I don't understand what the difference is here. The new line character or what?

Reply:
Yeah, the new editor config adds a new line at the end of the file. I made some changes to the file, then removed them, but am too lazy to revert the entire file (and the newline will inevitably be added eventually anyway).